docs/tutorials/multimodal/multimodal-faq.md

```diff
@@ -53,3 +53,21 @@ We used this script in our examples about [Petfinder Pawpularity](https://github
 and [Feedback Prize - Predicting Effective Arguments](https://github.com/autogluon/autogluon/tree/master/examples/automm/kaggle_feedback_prize). 
 You may refer to these examples for more details.
 
+## How does AutoGluon MultiModal handle multiple images per sample?
+
+We provide two options:
+1. Utilizing multiple image columns in a dataframe.
+2. Concatenating multiple image paths with semicolons and putting them into one dataframe column.
+
+These options can be used individually or in combination.
+During processing, we pass all images through a single image backbone and average their features and logits to obtain the final representation or prediction. 
+Note that for option 2, the maximum number of images allowed for one dataframe column can be controlled via hyperparameter `timm_image.max_img_num_per_col`, which has a default value of 2.
+
+## How does AutoGluon MultiModal handle multiple text columns in a dataframe?
+
+We tokenize each text field individually and then concatenate them into a single token sequence before feeding it into the model. 
```

Comment:
Ditto here

Reply:
Thanks. Added one sentence of introducing the auto-detection of all the text columns.